app/src/vuex/modules/account.js

```diff
@@ -35,12 +35,9 @@ export default ({ apollo }) => {
           localStorage.setItem(`auth_token`, idToken)
           // make sure new authorization token get added to header
           apollo.cache.reset()
-
-          console.log("User is now signed in!")
         } else {
           commit(`userSignedIn`, false)
           commit(`setUserInformation`, null)
-          console.log("User is now signed out!")
```

Comment:
This was helping the debugging :sweat_smile:

Reply:
yeah but we shouldn't leave console.logs in the production app